docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
See my comment here: https://github.com/kubernetes/kubernetes.github.io/pull/1265/files#r80270865

To summarize, I think the clarification re: addons vs the "/cluster/addons" directory is much better!  I still think we could improve the rest of the page so that it comes across less like marketing.